### PR TITLE
vdk-server: logs url in quickstart vdk

### PR DESCRIPTION
**Pull request by murphp15**
# Why
In quickstart vdk we want to be able to see the logs of a job. 
# What 
Read the logs from the api server
# How was this tested
Ran it locally. 

*The logs endpoint in the api is not brilliant. It doesn't return anything unless the job is complete*

images showing links in UI to links redirect link:  
Screenshot 2023-05-10 at 09 46 55
Screenshot 2023-05-10 at 09 47 01

image showing logs for job not yet started:
Screenshot 2023-05-10 at 09 44 39

image showing logs for completed job:
Screenshot 2023-05-10 at 09 44 45

### CODE REVIEW

File: projects/vdk-plugins/vdk-server/src/vdk/plugin/server/installer.py [hunk 1]
```diff
@@ -646,6 +646,8 @@ def __helm_install_command(self, git_server_ip):
             "--set",
             "deploymentBuilderResourcesDefault.limits.cpu=0",
             "--set",
+            "dataJob.executions.logsUrl.urlTemplate=http://localhost:8092/data-jobs/for-team/{{team_name}}/jobs/{{job_name}}/executions/{{execution_id}}/logs?tail_lines=400",
```
Comment (tozka) [inline projects/vdk-plugins/vdk-server/src/vdk/plugin/server/installer.py:649]: Why is that needed?

Comment (murphp15) [reply projects/vdk-plugins/vdk-server/src/vdk/plugin/server/installer.py:649]: in quickstart vdk we want ppl to be able to see the logs as provided by our api?

Comment (ivakoleva) [reply projects/vdk-plugins/vdk-server/src/vdk/plugin/server/installer.py:649]: nit: it just seems we have a lot of hardcoded duplicates of "http://localhost:8092" strings

**Review by ivakoleva — APPROVED** [review]
You can provide screenshot of how the link gets visualized, and how it looks like in the browser, to provide a quick context of the change

**Review by DeltaMichael — APPROVED** [review]
Regarding the endpoint for logs, I'm not sure if streaming live logs was scoped in the past. This might be a different issue entirely. We need further discussion here.

**Comment by murphp15** [comment]
@ivakoleva @DeltaMichael 
I updated the VEP and created the following two tickets: https://github.com/vmware/versatile-data-kit/issues/2041, https://github.com/vmware/versatile-data-kit/issues/2042.
I am going to merge this now.
